.github/ISSUE_TEMPLATE.md

```diff
@@ -1,31 +1,31 @@
+<!--
 If you encounter a bug or you have a support question, please try to fill out some of the information below.
-However, if you think your issue does not need any of it, you may omit it.
-Generally speaking, the information below is meant for helping debugging issues
-but they are no prerequisite for opening issues.
-
-### Subject of the issue
-Describe your issue here.
+Generally speaking, the information below is meant to help debugging issues but is no prerequisite for opening an issue.
```

Comment:
I would keep the `However...` to "allow" users delete entirely the template

Reply:
I think the users that want to delete the template will do it anyways ;)
And saying it is "no prerequisite for opening an issue" basically implies that. I'd err on the side of nudging people to use the template.

Reply:
Fair enough, thanks again for improve it